pkgs/tools/wayland/hyprevents/default.nix

```
@@ -0,0 +1,40 @@
+{ fetchgit, lib, makeWrapper, stdenv, hyprland, bash }:
+
+stdenv.mkDerivation rec {
+  pname = "hyprevents";
+  version = "1.0-20221209";
+
+  src = fetchgit {
```

Prefer `fetchFromGitHub` over `fetchgit`

```suggestion
  src = fetchFromGitHub {
```